### ISSUE TITLE
piqilib doesn't work in toplevel

### ISSUE THREAD

**Issue by ivg** [alavrik/piqi#50]
`piqirun.ext` doesn't work from toplevel:
```
# #use "topfind";;
- : unit = ()
Findlib has been successfully loaded. Additional directives:
  #require "package";;      to load a package
  #list;;                   to list the available packages
  #camlp4o;;                to load camlp4 (standard syntax)
  #camlp4r;;                to load camlp4 (revised syntax)
  #predicates "p,q,...";;   to set these predicates
  Topfind.reset();;         to force that packages will be reloaded
  #thread;;                 to enable threads

- : unit = ()
# #require "piqirun.ext";;
/home/ivg/.opam/4.02.1/lib/ulex: added to search path
/home/ivg/.opam/4.02.1/lib/ulex/pa_ulex.cma: loaded
/home/ivg/.opam/4.02.1/lib/ulex/ulexing.cma: loaded
/home/ivg/.opam/4.02.1/lib/easy-format: added to search path
/home/ivg/.opam/4.02.1/lib/easy-format/easy_format.cmo: loaded
/home/ivg/.opam/4.02.1/lib/xmlm: added to search path
/home/ivg/.opam/4.02.1/lib/xmlm/xmlm.cma: loaded
/home/ivg/.opam/4.02.1/lib/bytes: added to search path
/home/ivg/.opam/4.02.1/lib/base64: added to search path
/home/ivg/.opam/4.02.1/lib/base64/base64.cma: loaded
/home/ivg/.opam/4.02.1/lib/piqilib: added to search path
# #require "piqirun.ext";;/piqilib/piqilib.cma: loaded
Error: The external function `camlidl_piqi_c_piqi_strtoull' is not available
# 
```

The issue manifests at least for `0.6.12`

**Comment by KennethAdamMiller** [comment]
I realized that I have several commits on different projects that never made it out into merge requests.

**Comment by ivg** [comment]
The problem is that [this change][1] is not in opam:

```
$ opam source piqilib
$ less piqilib.0.6.12/piqilib/Makefile | grep STATIC
STATIC = 1 # force creation of static library (don't create dlls)
```

[1]: https://github.com/alavrik/piqi/commit/c17f83de1870ccf73bbdd19c440350ed6d9c806b

**Comment by KennethAdamMiller** [comment]
I re-opened my laptop in the few minutes just before I leave to take a friend back to the airport and went to tell you I remembered that it had been made a merge request and the other stuff, but literally you had already found that out, ha :)

**Comment by alavrik** [comment]
@ivg I'll cut a new stable version and push it to opam in the next couple of days. sorry about the delay.

**Comment by alavrik** [comment]
the fix has been released as a part of piqilib-0.6.13
